Makefile

```diff
@@ -81,7 +97,7 @@ test-skopeo:
 		cd $${skopeo_path} && \
 		GO111MODULE="on" go mod edit -replace $${project_module}=$${project_path} && \
 		make vendor && \
-		make BUILDTAGS="$(BUILDTAGS)" binary-local test-all-local && \
+		make BUILDTAGS="$(BUILDTAGS)" binary-local test-unit-local && \
```

Comment:
Doesn’t removing the Skopeo integration + system tests pretty much defeat the point of `test-skopeo`? What tests that the code works against real-world registries now?

Reply:
`test-all-local` is triggering `validate-local` and `test-unit-local` with
```Makefile
validate-local:
        hack/make.sh validate-git-marks validate-gofmt validate-lint validate-vet
```
Maybe I am missing something but I don't see the integration tests being triggered by `make.sh` as we specify arguments.

Reply:
Oh, you’re right, sorry. The important tests are run in `make check` one line below, I just reacted to the `all` word without reading the details.

(I’m still mildly curious why `validate-local` broke, but it’s not nearly as important.)